change_column_type_materialized_view/README.md

```diff
@@ -67,14 +67,24 @@ SQL >
         pathname
 
 TYPE COPY
-DATASOURCE analytics_pages_mv_1
+TARGET_DATASOURCE analytics_pages_mv_1
 ```
 
 ## 2: Run CI
 
 Make sure the changes are deployed correctly in the CI Tinybird Branch. Optionally you can add automated tests or verify it from the `tmp_ci_*` Branch created as part of the CI pipeline.
 
-## 3: Backfilling 
+## 3: (For large datasets) Splitting the Data into Chunks for Backfilling
+
+If your data source is large, you may run into a memory error like this:
+```
+error: "There was a problem while copying data: [Error] Memory limit (for query) exceeded. Make sure the query just process the required data. Contact us at support@tinybird.co for help or read this SQL tip: https://tinybird.co/docs/guides/best-practices-for-faster-sql.html#memory-limit-reached-title"
+```
+
+To avoid memory issues, you will need to break the backfill operation into smaller, manageable chunks. This approach reduces the memory load per query by processing only a subset of the data at a time. You can use the data source's sorting key to define each chunk.
+
```

Comment:
Should we just link this guide? => https://www.tinybird.co/docs/work-with-data/strategies/backfill-strategies#scenario-3-streaming-ingestion-with-incremental-timestamp-column

Reply:
In addition to linking that guide, I would make sure that the field that they are using to split the data source into chunks is a sorting key.

Reply:
@alrocar @ramiroaznar Thanks for reviewing! Made those changes.

Reply:
Thanks!